### PR TITLE
snoop, truss, locale, bugreportz, dumpsys: fix linter issues

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me

**Review by kbdharun — APPROVED** [review]

**Comment by Magrid0** [comment]
@kbdharun is this PR ready to be merged?

**Comment by kbdharun** [comment]
> @kbdharun is this PR ready to be merged?

I would suggest waiting for @reinhart1010's review to check the change to the Indonesian pages, as I don't see this wording being used in other SunOS pages.

File: pages.id/sunos/snoop.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # snoop
 
 > Pengendus paket jaringan.
-> tcpdump untuk SunOS.
+> SunOS setara dengan tcpdump.
```
Comment (reinhart1010) [inline pages.id/sunos/snoop.md:4]: No, this significantly change the context from `the SunOS equivalent to the tcpdump command` to become `this SunOS platform is the same as tcpdump`.

I think it’s okay to use the current translation.

**Review by reinhart1010 — DISMISSED** [review]

File: pages.id/sunos/snoop.md (hunk 1 above)
Comment (kbdharun) [reply pages.id/sunos/snoop.md:4]: Aren't we ignoring this rule here/restricting it to English pages? 

Maybe we can update tldr-maintenance to do the same i.e. only check this for English pages.

Comment (reinhart1010) [reply pages.id/sunos/snoop.md:4]: > > I think it’s okay to use the current translation.
> 
> It isn't since that is causing a linter issue. Could you rephrase the sentence in a way that it starts with a capital letter?

Well it could be

```suggestion
Perintah setara tcpdump untuk SunOS.
```
